src/Tools/Github/GitMergeBot/project.json

```diff
@@ -1,7 +1,7 @@
 {
   "dependencies": {
     "LibGit2Sharp": "0.22.0",
-    "LibGit2Sharp.NativeBinaries": "1.0.160",
+    "LibGit2Sharp.NativeBinaries": "1.0.129",
```

Comment:
Why don't we just delete this? LibGit2Sharp already has the dependency on the version it needs. Explicitly listing it is just making work for ourselves.